image.md

```diff
@@ -0,0 +1,309 @@
+# Image
+
+This section defines a format for encoding a container's *filesytem bundle* as a portable *image*.
+An image is a set of files organized in a certain way, containing all of the necessary data and metadata, for any compliant runtime to verify the validity of the image's contents and to construct a filesystem bundle represented by the image.
+Images are meant to be portable such that they can be used to reliably move filesystem bundles between instances of OCI compliant implementations.
+
+In its simplest form, an image is a tar file of the filesystem bundle (root file system directory plus a configuration file) along with some additional metadata.
+While the runtime only sees a single directory as the root filesystem, it might actually be stored as a layering of filesytems under the covers.
+When representing this layering within an image, each "layer" will be independently referenced.
+
+## Image Contents
+
+A compliant image contains the following pieces of information:
+
+1. A `manifest.json` file describing the contents of the image.
+This file is REQUIRED to be present in the image.
+This file extends the [`config.json`](config.md) file with some additional properties:
```

Comment:
I'm struggling on this.
As the `config.json` has evolved to essentially be a instance configuration for a container, I don't see how the `manifest.json` is particularly an extension. This comes back to the F2F conversations about having templates, or "suggests" or "requests" for resources. But the config.json is really not portable and would come with a list of caveats and exceptions, right?